node/src/chain_spec.rs

```diff
@@ -468,5 +471,35 @@ fn lerna_genesis(
 		}),
 		pallet_elections_phragmen: Some(ElectionsConfig { members: vec![] }),
 		pallet_collective_Instance1: Some(CouncilConfig::default()),
+		pallet_claims: Some(ClaimsConfig { claims: vec![] }),
 	}
 }
+
+fn create_testnet_claims() -> Vec<(EthereumAddress, Balance)> {
+	let mut claims = Vec::<(EthereumAddress, Balance)>::new();
+
+	// Alice's claim
+	// Signature: 0xbcae7d4f96f71cf974c173ae936a1a79083af7f76232efbf8a568b7f990eceed73c2465bba769de959b7f6ac5690162b61eb90949901464d0fa158a83022a0741c
+	// Message: "I hereby claim all my HDX tokens to wallet:d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"
+	let claim_address_1 = (
+		EthereumAddress(hex!["8202c0af5962b750123ce1a9b12e1c30a4973557"]),
+		50_000_000_000_000_000,
+	);
+
+	// Another claim
+	let claim_address_2 = (
```

Comment:
Remove these before merging

Reply:
Richard just added some additional info like private keys to claim2 & claim3, remove anyway?

Reply:
The question is What data do we want to use for claims in local testnets?

Reply:
Ok if we have priv keys we're good. But add some reasonable amounts to them then. 
0.001, 1, and 1000 * 10^12 tokens should be good?

Reply:
Is 1 equal to 10^12 ?

Reply:
Yes.

Here:
https://github.com/galacticcouncil/HydraDX-node/blob/496178440b9ed980d58ab07cecfc3c0432a7e877/runtime/src/constants.rs#L4

And here:
https://github.com/galacticcouncil/HydraDX-node/blob/fadfe28ff04abf504caa0adea475d2a3e04c82d8/node/src/chain_spec.rs#L87